File: change_dept.php
<?php


//work out fy from selected year

$dept_id = $_REQUEST['dept_id'];
$value	 = $_REQUEST['value'];


include_once("configi.php");

		$sqlupdate1 = "update web_dept set graph_sel = ".$value." where dept_id=".$dept_id;
		
	//	echo "<br>".$sqlupdate1."<br>";
		
	
		$resultinsert1=mysqli_query($conn, $sqlupdate1);


echo "successful";
	
	
exit;
	

File: changedept.py
#!C:\Users\Akasha\AppData\Local\Programs\Python\Python39-32\python.exe

import sys
import requests
import pyodbc 
import time


#while True:


dept_id = sys.argv[1]
dept_name = sys.argv[2]

run_dir = r'D:/holistic/bookshop/bookdata.accdb'

#print (run_dir)
#connection srings C:\Users\Akasha\AppData\Local\Programs\Python\Python39-32\python.exe C:\Users\davidmarsh\AppData\Local\Programs\Python\Python39-32\python.exe
    
connStr = (
        r"DRIVER={Microsoft Access Driver (*.mdb, *.accdb)};"
        r"DBQ="+run_dir+";"
        )

conn = pyodbc.connect(connStr)

cursor = conn.cursor()  

cursor.execute("update web_dept set dept_name = ? where dept_id =?",dept_name, dept_id)

cursor.commit()

cursor.close()

print ("successful")


